src/WorldGenerator.js

```diff
@@ -7,10 +7,27 @@ var WorldGenerator = function () {
 WorldGenerator.prototype = {
   generateWorld: function(){
     console.log("generator started!");
-
+    noise.seed(Math.random());
     planetData.generateMap();
     game.cache.addBitmapData('randomTex', randomTexture());
-    // Put generation code in here
+    /* generate a  bitmap to use as a fancy background image in the level */
+    /* image is a quarter of the screen size because we do not need high-res */
+    var bmd = game.make.bitmapData(window.game.width * 0.5, window.game.height * 0.5);
```

Comment:
We aren't going to use window to access the game object so this should just use `game.width` and `game.height`.
I just noticed now that we've let this through in other places. If you don't want to fix those up here just let me know and I can do it in a separate pull request.